corehq/motech/fhir/utils.py

```diff
@@ -53,3 +58,15 @@ def update_fhir_resource_property(case_property, fhir_resource_type, fhir_resour
                                                       resource_type=fhir_resource_type)
         fhir_resource_prop.jsonpath = fhir_resource_prop_path
         fhir_resource_prop.save()
+
+
+@memoized
+def load_fhir_resource_types(fhir_version=FHIR_VERSION_4_0_1, exclude_resource_types=('fhir',)):
+    schemas_base_path = get_schema_dir(fhir_version)
+    all_file_names = glob.glob(os.path.join(schemas_base_path, '*.schema.json'))
```

Comment:
nit: You could consider using `os.listdir(schemas_base_path)` that would make this simpler.

Reply:
Wouldn't `os.listdir(schemas_base_path)` include files that don't end in ".schema.json" -- i.e. are not schemas?

Reply:
Noted. Makes sense. Will use the listdir here.

Reply:
Right, I didn't specify but we should ignore files that don't end with ".schema.json".

> just returns filenames

Yup!